build-system/tasks/e2e/amp-driver.js

```diff
@@ -14,6 +14,51 @@
  * limitations under the License.
  */
 
+
+/** @enum {string} */
+const Environment = {
+  SINGLE: 'single',
```

Comment:
Hmm, there's a precedent for `single` as an `ampdoc` environment.

https://github.com/ampproject/amphtml/blob/e194f8e0726b8a3241dcac2001c43669a3df88ae/extensions/amp-ad/0.1/test/test-amp-ad-custom.js#L141-L145

Would the precedent make you feel better about it?

Reply:
Gotcha, but here the value `single` is associated with the property `ampdoc` and so it reads well. In your case you're associating `single` with an `environment`, which is not intuitive to me. Though the following would be clearer.
    
    const AMP_DOC_ENVIRONMENT = {
      SINGLE: 'single',
    }

then it would be clearer. I don't feel strongly about this. Perhaps the fact that others have not pointed this out probably shows that I am the only one not too keen about the naming.